protocol/v2/qbft/instance/instance.go

```diff
@@ -145,12 +145,18 @@ func (i *Instance) SetConfig(config types.IConfig) {
 
 // GetHeight interface implementation
 func (i *Instance) GetHeight() specqbft.Height {
-	return i.State.Height
+	if s := i.State; s != nil {
```

Comment:
@amir-blox @olegshmuelov in which case this is happening?
is it relevant to spec as well?

Reply:
@lior-blox changed the fix https://github.com/bloxapp/ssv/pull/753/commits/44410e9aae2e697cc485f186ed0d0fe9183b3b42